rhizome/host/bin/setup-nftables.rb

```diff
@@ -0,0 +1,49 @@
+#!/bin/env ruby
+# frozen_string_literal: true
+
+require_relative "../../common/lib/util"
+require "fileutils"
+require "json"
+
+unless (additional_ip_addresses = ARGV.shift)
+  puts "additional IP addresses didn't get passed"
+  additional_ip_addresses = "[]"
+end
+
+# setup blocking unused ip addresses
+NFTABLES_PATH = "/etc/nftables.d/0.conf"
+FileUtils.mkdir_p("/etc/nftables.d")
+FileUtils.touch(NFTABLES_PATH)
+ip_ranges_to_block = JSON.parse(additional_ip_addresses)
+
+safe_write_to_file(NFTABLES_PATH, <<SETUP_ADDITIONAL_IP_BLOCKING)
+#!/usr/sbin/nft -f
+table inet drop_unused_ip_packets;
+delete table inet drop_unused_ip_packets;
+table inet drop_unused_ip_packets {
+  set allowed_ipv4_addresses {
+    type ipv4_addr;
+  }
+
+  set blocked_ipv4_addresses {
+    type ipv4_addr;
+    flags interval;
+    elements = {#{ip_ranges_to_block.join(",")}}
+  }
+
+  chain prerouting {
+    type filter hook prerouting priority 0; policy accept;
+    ip daddr @allowed_ipv4_addresses accept
+    ip daddr @blocked_ipv4_addresses drop
+  }
+}
+SETUP_ADDITIONAL_IP_BLOCKING
```

Comment:
So, my understanding it the config files included later on overwrites the previous one. Is that correct? How the order of inclusion is decided? Is it alphabetical?

Reply:
Yes, it's alphabetical. Hence the base rule to have the name 0.conf;
From the manual of nftables;
```
The wildcard matches are loaded in alphabetical order. 
```
[Nftables manual](https://www.netfilter.org/projects/nftables/manpage.html)